factcast-core/src/main/java/org/factcast/core/FactCast.java

```diff
@@ -80,7 +80,8 @@ default LockedOperationBuilder lock(@NonNull FactSpec scope, FactSpec... tail) {
     @Deprecated
     DeprecatedLockedOperationBuilder lock(@NonNull String ns);
 
-    // removed from interface as this breaks the updated locking code and
+    // removed from interface as this breaks the updated locking code and //
+    // TODO maybe must be removed from interface as this breaks the updated
     // was a bad idea in the first place. See migration guide (0.3.0)
     //
     // DeprecatedLockedOperationBuilder lockGlobally();
```

Comment:
Delete alltogether?

Reply:
i leave this in here as a trace for desperate people hinting (AGAIN) to the migration guide :D

Reply:
fixed the c&p error